src/panic-defer-recover/defer.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+func startJob() {
+	//Defer is used to ensure that a function call is performed later
```

Comment:
since the purpose is educational, I'd be more specfiic here; It's not only "later" but when the function returns to the caller